### PR TITLE
Announcing some Symfony conferences 2024

### PR DESCRIPTION
**Pull request by GromNaN**
- [SymfonyOnline January 2024](https://live.symfony.com/2024-online-january/)
- [SymfonyLive Paris 2024](https://live.symfony.com/2024-paris/)
- [SymfonyCon Vienna 2024](https://live.symfony.com/2024-vienna-con/)

### CODE REVIEW

File: archive/entries/2024-01-15-1.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>SymfonyOnline January 2024</title>
+  <id>https://www.php.net/archive/2024.php#2024-01-15-2</id>
```
Comment (javiereguiluz) [inline archive/entries/2024-01-15-1.xml:4]: There's a mismatch between the ID in the URL here and the ID in the file name.

File: archive/entries/2024-01-15-2.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>SymfonyLive Paris 2024</title>
+  <id>https://www.php.net/archive/2024.php#2024-01-15-1</id>
```
Comment (javiereguiluz) [inline archive/entries/2024-01-15-2.xml:4]: Same here. The URL should be `https://www.php.net/archive/2024.php#2024-01-15-2`

File: archive/entries/2024-01-15-1.xml (hunk 1 above)
Comment (GromNaN) [reply archive/entries/2024-01-15-1.xml]: Oups, fixed both.

**Review by sy-records — APPROVED** [review]

**Comment by GromNaN** [comment]
The conferences doesn't showup on the preview website: https://web-php-pr-924.preview.thephp.foundation/archive/2024.php

I'm not sure if it should work.

**Review by saundefined — APPROVED** [review]
Thank you!

**Pull request merged**